### ISSUE TITLE
Streams Mixed??

### ISSUE THREAD

**Issue by ksclark** [kilianc/rtail#116]
When running the following commands.

tail -f /var/webapps/eventscheduler.log | rtail --id 'test0'
tail -f /var/webapps/tracker.log | rtail --id 'test1'

If I click on stream 'test1' and then click on stream 'test0' or vice versa.  The log data from both files appear in both streams in the web app.

Any ideas?

**Comment by kilianc** [comment]
This shouldn't be happening. Are you 100% sure you are not writing the two files with the same content?

OS / Node version / rtail version?

**Comment by maaronking** [comment]
I am seeing the same issue.  If stay on one stream, everything looks correct.  However, if I click on another stream and then back again, it mixes the two streams.

```
prompt> rtail --version
0.2.1
```

```
prompt> rtail-server --version
0.2.1
```

```
prompt> node --version
v0.10.25
```

```
prompt> uname -a
Linux ubuntu 3.16.0-69-generic #89~14.04.1-Ubuntu SMP Thu Mar 17 20:51:04 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

```

**Comment by maaronking** [comment]
Not sure if this is related, but the web console prints `test: backlog received 100
bundle.min.js:18` each time i switch back and forth to various streams (obviously where 'test' is the name of the stream that I switched to).

**Comment by maaronking** [comment]
Also, if you switch to a stream and start seeing logs from the previous, if you refresh the page, you only see the logs from the current stream again.

**Comment by kilianc** [comment]
Yeah I think something changed somewhere in the deps tree or in the browser API because it used to work. I am def need to look into it. Thanks for reporting and adding more info, it's very helpful.

**Comment by demobox** [comment]
Seeing the same issue - if there's any data we can provide that can help chase this one down, please let me know!
